src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -107,14 +129,25 @@ size_t DBSCAN<RangeSearchType, PointSelectionPolicy>::Cluster(
 
   // Now set assignments.
   assignments.set_size(data.n_cols);
+  #pragma omp parallel for
   for (size_t i = 0; i < data.n_cols; ++i)
     assignments[i] = uf.Find(i);
 
   // Get a count of all clusters.
-  const size_t numClusters = max(assignments) + 1;
+  const size_t numClusters = arma::max(assignments) + 1;
```

Comment:
any reason to add the `arma`, we removed this intentionally, so if there is no error could you remove it please?